.gitignore

```diff
@@ -51,3 +51,4 @@ pkg/loki/wal
 
 # nix
 result
+/docs/make-docs
```

Comment:
To avoid network calls in CI to fetch this file, I think we should remove the ignore and commit this file. WDYT?